src/EditorFeatures/CSharpTest2/Recommendations/DelegateKeywordRecommenderTests.cs

```diff
@@ -43,16 +44,16 @@ public async Task TestAfterGlobalVariableDeclaration_Interactive()
         }
 
         [Fact, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
-        public async Task TestNotInUsingAlias()
+        public async Task TestInUsingAlias()
         {
-            await VerifyAbsenceAsync(
+            await VerifyKeywordAsync(
 @"using Goo = $$");
```

Comment:
wait why?  i would still expect these to *not* be offered here.

Reply:
Why not? You can alias them.

Reply:
Really?  We don't let you alias tuples, or pointers or arrays...  But we let you alias a funcptr??

Reply:
Hmm, I thought we did, but apparently we do not.

Reply:
Adjusted.